### ISSUE TITLE
HTML in UiSnackbar

### ISSUE THREAD

**Issue by nivv** [JosephusPaye/Keen-UI#75]
I'm using the snackbar to display different messages and I create them dynamically. 

```javascript
    watch: {
        'snackbarMessage': function(message) {
            this.$broadcast('ui-snackbar::create', {
                message: message.message,
                duration: message.duration
            });
        }
    }
```
The problem with this approach is that I'm not sure how to put the message in the `slot`, because the slot supports HTML but the `message` does not. 

Is there a way around this?

Also, would be great if we could add an icon to the snackbar message as well.

**Comment by JosephusPaye** [comment]
Right now, the only way around this would be customizing UiSnackbarContainer to bind the `message` string as HTML.

So the template in [UiSnackbarContainer](https://github.com/JosephusPaye/Keen-UI/blob/master/src/UiSnackbarContainer.vue#L1-L13) would be changed to:

```html
<template>
    <div class="ui-snackbar-container" :class="[position]">
        <ui-snackbar
            :duration="s.duration" :show.sync="s.show" :action="s.action" auto-hide
            :action-color="s.actionColor" :persistent="s.persistent" :id="s.id"

            @shown="shown(s)" @hidden="hidden(s)" @clicked="clicked(s)"
            @action-clicked="actionClicked(s)"

            v-for="s in queue"
        >
            <div class="my-custom-snackbar-message" v-html="s.message"></div>
       </ui-snackbar>
    </div>
</template>
```

You can make a custom component by copying everything in UiSnackbarContainer and making the change above.

This is only a temporary solution as HTML in `message` will be supported in the next version. With HTML supported, you can then use icons or anything else you need in the content. 

However, the [Material Design spec](https://material.google.com/components/snackbars-toasts.html#snackbars-toasts-usage) advises against using icons in the snackbar.

**Comment by nivv** [comment]
Cool! Not a huge issue for me as it is now. But nice to know that it will be added. What I would want to do is to display a colored snackbar depending on success or error and stuff like that.

**Comment by JosephusPaye** [comment]
Now available in [v0.8.9](https://github.com/JosephusPaye/Keen-UI/releases/tag/v0.8.9), with the new `allowHtml` prop on [UiSnackbarContainer](https://josephuspaye.github.io/Keen-UI/#/ui-snackbar-docs).
